pkg/kubelet/certificate/certificate_manager.go

```diff
@@ -156,6 +159,12 @@ func (m *manager) Current() *tls.Certificate {
 	return m.cert
 }
 
+// CertificateSigningRequestClient sets the client interface that is used for
+// signing new certificates generated as part of rotation.
+func (m *manager) CertificateSigningRequestClient(certSigningRequestClient certificatesclient.CertificateSigningRequestInterface) {
+	m.certSigningRequestClient = certSigningRequestClient
```

Comment:
This isn't thread safe. If it doesn't need to be since this must be called before `Start()`, then this should at least check that `m.certSigningRequestClient` is `nil` first.